template/page-docs.html.jinja

```diff
@@ -1,126 +1,429 @@
 {% extends "base.html.jinja" %}
 
-{% block mainclasses %}landing page-docs-index landing-builtin-bg{% endblock %}
+{% block mainclasses %}landing page-docs-index landing-builtin-bg jackson-identifier {% endblock %}
```

Comment:
I'm not sure why this isn't showing up locally for me - but `jackson-identifier` should probably be removed.